docs/topics/modules/planning_development_work.adoc

```diff
@@ -1,54 +1,29 @@
 [id="planning_development_work"]
 = Planning your development work
 
-When you create a space, the *Create an Application* wizard is displayed. Close the wizard, so that you can plan an end to end development workflow for launching and deploying a sample application before you launch the application.
+When you create a space, the *Create an Application* wizard is displayed. Close the wizard, so that you can plan an end to end development workflow before you launch the application.
 
-{osio} planner provides a set of predefined templates based on common development methodologies such as Agile and Scenario Driven Development. The development process you select while creating the space determines the planner's guided work item type hierarchy.
+{osio} planner provides a set of predefined templates based on common development methodologies such as Agile and Scenario Driven Development. The development process you select while creating the space determines the guided work item type hierarchy in the planner.
 
-This example walks you through the Scenario Driven Development planning process to plan your work.
+This example walks you through the Agile planning process to plan your work.
 
-. In the dashboard of your space, select the *Plan* tab at the top of the page to view the default *Backlog* view of the planner.
-. Select the *Show Tree* check box to see the work items and their children in a tree view. The tree view allows you to quickly add work items according to the guided hierarchy using the quick-add and in-line add options.
-. Use the quick-add option to create a *Scenario* as follows:
-.. In the *Add Work Item* pane, select *Scenario* from the drop-down list.
-+
-image::quickadd_scenario.png[Quick add scenario]
-+
-.. In the *Work Item Title* field, add *Enable an end to end, developer work-flow, using the continuous integration and deployment framework* as the title.
-.. Press kbd:[Enter] to create the scenario.
+. In the dashboard of your space, select the *Plan* tab at the top of the page. The default *Backlog* view displays the *Work Items* for your space.
+. Select the *Show Tree* check box to use the quick-add and in-line add options to quickly create workitems and their children.
```

Comment:
I was just testing these instructions and this was selected by default for me (fresh env, not used planner before here).

Reply:
Are you sure? Cause for me it did not, I checked yesterday, will check again.

Reply:
Yes, sorry, see above, I was in a SDD space. I'm now in an Agile one and I can see the options.